src/sprites/clone.js

```diff
@@ -200,15 +205,17 @@ Clone.prototype.setDirection = function (direction) {
  * @param {?string} message Message to put in say bubble.
  */
 Clone.prototype.setSay = function (type, message) {
-    if (this.isStage) {
+    if (this.isStage || !this.render) {
```

Comment:
do you mean `this.renderer`? I can't imagine this would work as I don't think such a variable exists?